DOCUMENTATION.md

```diff
@@ -29,6 +30,12 @@ This module implements all models for Resin Python SDK.
 ## Application
 
 This class implements application model for Resin Python SDK.
+
+Due to API changes, the returned Application object schema has changed. Here are the formats of the old and new returned objects.
+
+The old returned object's properties: `__metadata, actor, app_name, application, commit, device_type, git_repository, id, support_expiry_date, user, version, should_track_latest_release`.
+
+The new returned object's properties (since Python SDK v1.7.0): `__metadata, actor, app_name, commit, device_type, git_repository, id, is_accessible_by_support_until__date, should_track_latest_release, user, version, depends_on__application`.
```

Comment:
List should be:
__metadata, actor, app_name, commit, depends_on__application, device_type, git_repository, id, is_accessible_by_support_until__date, should_track_latest_release, user, version